src/plugins/template/src/config.cpp

```diff
@@ -83,6 +83,50 @@ Configuration::Configuration(const ov::AnyMap& config, const Configuration& defa
             log_level = value.as<ov::log::Level>();
         } else if (ov::hint::model_priority == key) {
             model_priority = value.as<ov::hint::Priority>();
+        } else if (ov::optimal_batch_size == key) {
+            try {
+                int tmp_i = value.as<int>();
+                if (tmp_i >= 0)
+                    optimal_batch_size = tmp_i;
+                else
+                    OPENVINO_THROW("Incorrect value, it should be unsigned integer: ", key);
+            } catch (const std::exception&) {
+                OPENVINO_THROW("Wrong value ", value.as<std::string>(), "for property key ", key);
+            }
+        } else if (ov::max_batch_size == key) {
+            try {
+               int tmp_i = value.as<int>();
+                if (tmp_i >= 0)
+                    max_batch_size = tmp_i;
+                else
+                    OPENVINO_THROW("Incorrect value, it should be unsigned integer: ", key);
+            } catch (const std::exception&) {
+                OPENVINO_THROW("Wrong value ", value.as<std::string>(), "for property key ", key);
+            }
+        } else if (ov::loaded_from_cache == key) {
+            loaded_from_cache = value.as<bool>();
+        } else if (ov::enable_mmap == key) {
+            enable_mmap = value.as<bool>();
```

Comment:
`ov::enable_mmap` is only for frontends, I suppose it won't be supported by plugins

Reply:
`ov::enable_mmap` is not used for plugin, reference to:
https://github.com/openvinotoolkit/openvino/blob/f55e967dd5a65d2f7f48f8e1197f20f824b7fe28/src/inference/include/openvino/runtime/properties.hpp#L743-L752

Reply:
removed cases for enable_mmap
I also removed test cases, let me know if it is correct or not.